docs/guides/tipsntricks.rst

```diff
@@ -114,3 +114,22 @@ Test generation (testgen)
 -------------------------
 
 TODO
+
+Working with file paths
+-----------------------
+For relative file pathnames in the project, there are several helper functions that can be used:
+
+* utils.path.project_path for project root (e.g. cfme_tests)
+* utils.path.conf_path for configuration directory (e.g. cfme_tests/conf)
+* utils.path.data_path for data directory (e.g. cfme_tests/data)
+* utils.path.docs_path for documentation directory (e.g. cfme_tests/docs)
+* utils.path.log_path for logfile directory (e.g. cfme_tests/log)
+* utils.path.scripts_path for scripts directory (e.g. cfme_tests/scripts)
+* utils.path.scripts_data_path for scripts data directory (e.g. cfme_tests/scripts/data)
+* utils.path.template_path for template directory (e.g. cfme_tests/template)
```

Comment:
I don't want to be on the hook to maintain the list, and I think the autogenerated one on the `utils.path` module page is good enough (or could be made good enough if the docstrings in the module need improvement).

`utils.path` works for all paths (including absolute ones) relative to the project root, not just relative file paths in the project root. (i.e. change "For relative file pathnames in the project root" to "For any path in the project root").